client/components/EditProfile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+export const EditProfile = (props) => {
+
+  return (
+    <div>
+      <form>
```

Comment:
is there functionality in this form yet?

Reply:
no not yet - i got sidetracked with the search button functionality on the front page